Components/input.css

```diff
@@ -0,0 +1,25 @@
+.input{
+    display: flex;
+    flex-direction: column;
+    padding:15px;
+    width:30%
+}
+input:focus{
+    outline:none;
+}
+
+.username{
```

Comment:
instead of .username, you can name it **form-input-username**, **form-input-password** or something easy to understand.